entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+source /opt/ros/noetic/setup.bash
+[ -f /opt/barracuda-dvl/catkin_ws/devel/setup.bash ] && source /opt/barracuda-dvl/catkin_ws/devel/setup.bash
+exec /bin/bash
```

Comment:
exec tail -f /dev/null is standard for keeping a container open. but in the future this should be replaced with a launch file that runs whatever node that needs to get started. be sure to use the --wait flag with roslaunch so that you don't automatically start a roscore.